data_gatherer.py

```diff
@@ -64,17 +68,17 @@ def main(argv):
         control_loop_stream, release_sensor_stream,
         pipeline_finish_notify_stream)
 
-    # Add sensors.
-    (center_camera_stream, notify_rgb_stream,
-     rgb_camera_setup) = pylot.operator_creator.add_rgb_camera(
-         transform, vehicle_id_stream, release_sensor_stream)
-    (depth_camera_stream, _,
-     depth_camera_setup) = pylot.operator_creator.add_depth_camera(
-         transform, vehicle_id_stream, release_sensor_stream)
-    (segmented_stream, _,
-     _) = pylot.operator_creator.add_segmented_camera(transform,
-                                                      vehicle_id_stream,
-                                                      release_sensor_stream)
+    # TODO: Define a function that ensures consistency of this flag.
```

Comment:
I would clean up this so that a camera is only added if log_rgb_camera is true, depth camera if log_depth camera is true, and similarly for segmented. In this way, you're not going to need the additional skip_cameras flag.